packages/marketing-modal/src/MarketingModal/types.ts

```diff
@@ -0,0 +1,70 @@
+import { ModalProps } from '@leafygreen-ui/modal';
+
+export const BlobPosition = {
+  TopLeft: 'top left',
+  TopRight: 'top right',
+  BottomRight: 'bottom right',
+} as const;
+
+export type BlobPosition = typeof BlobPosition[keyof typeof BlobPosition];
+
+export const GraphicStyle = {
+  Center: 'center',
+  Fill: 'fill',
+} as const;
+
+export type GraphicStyle = typeof GraphicStyle[keyof typeof GraphicStyle];
+
+export interface MarketingModalProps extends ModalProps {
+  /**
+   * Text of header element
+   */
+  title: string;
+  /**
+   * React Element to be rendered as the modal's hero image
+   */
+  graphic: React.ReactElement;
+  /**
+   * Determines the rendering style of the graphic.
+   *
+   * `fill` adds a curving effect to the bottom border of the graphic.
+   */
+  graphicStyle?: GraphicStyle;
+  children: React.ReactNode;
```

Comment:
does this need a tsdoc definition?

Reply:
I don't think we even need to define this since this is inherited from `ModalProps`. I'm gonna remove it.